test/verify/check-users

```diff
@@ -23,6 +23,39 @@ import parent  # noqa: F401
 from testlib import MachineCase, nondestructive, skipDistroPackage, skipImage, todoPybridge, test_main, wait
 
 
+good_password = "tqymuVh.ZfZnP§9Wr=LM3JyG5yx"
+
+
+def performUserAction(browser, user, action):
```

Comment:
Can you please send this "split test-accounts" into a separate PR and already land it? If that does not have functional changes, that's hopefully quick. Otherwise, if it exposes flakes, they may be easier to investigate with just that change instead of all 8 commits here.

Reply:
Ah, didn't notice this comment until now.  Done: https://github.com/cockpit-project/cockpit/pull/18349